### ISSUE TITLE
error initializing fresh database with 0.23.0

### ISSUE THREAD

**Issue by strowi** [fnep/mtv_dl#66]
Hi,

i just ran into this issue on a clean pi4 with v0.23.0:

```bash
~> mtv_dl list -r title='Letzte Spur'
Downloading database 
....
━━━━━━━━━━━━━━━━━━━━━━━╺ 99.6% 0:00:02Reading database items ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100.0% 0:00:00
2023-06-02T21:15:05  INFO     Initializing History database in PosixPath('/home/osmc/mtv/.Filmliste.0.23.0.sqlite').
2023-06-02T21:15:05  CRITICAL <class 'sqlite3.OperationalError'>: near "(": syntax error
                                File "/usr/local/bin/mtv_dl", line 8, in <module>
                                  sys.exit(main())
                                File "/usr/local/lib/python3.9/dist-packages/mtv_dl.py", line 1461, in main
                                  showlist = Database(filmliste=cw_dir / FILMLISTE_DATABASE_FILE, history=cw_dir / HISTORY_DATABASE_FILE)
                                File "/usr/local/lib/python3.9/dist-packages/mtv_dl.py", line 473, in __init__
                                  self.initialize_history()
                                File "/usr/local/lib/python3.9/dist-packages/mtv_dl.py", line 420, in initialize_history
                                  cursor.execute(
```

Deleting the sqlite-files, downgrading to v0.22.0  and running the command again works. And after that i can upgrade to v0.23.0 which shows an additional info 

`2023-06-02T21:40:10  INFO     Upgrading history database schema, adding columns for season and episode`

 and then it succeeds.

Looks to me as if sth. changed in the initialisation of the db?

**Comment by fnep** [comment]
Thank you for reporting this. I already have an idea what it is and will have a look.

**Comment by fnep** [comment]
I uploaded a fixed version as [v0.23.2](https://pypi.org/project/mtv-dl/0.23.2/).

**Comment by strowi** [comment]
Thank you for the quick work. Confirmed working. ;)
